pre-registration-default.properties

```diff
@@ -191,6 +196,7 @@ mosip.kernel.masterdata.validdoc.rest.uri=${masterdata.resource.url}/validdocume
 ##Kernel Service URL for fetching templates
 resource.template.url=${masterdata.resource.url}/templates
 mosip.pre-registration.notification.id=mosip.pre-registration.notification.notify
+mosip.optional-languages=fra,ara,hin,kan,tam
```

Comment:
Why are we configuring this again here, will it not take from application-default.properties ?

Reply:
I think this was added by Aiham to support the API test rig.

Reply:
In that case, we don't need this change in the release branch ?

Reply:
@dhanendra06 to check with @mayuradesh and remove this change

Reply:
@mayuradesh confirmed we can remove these changes, but the API test rig may not work so removing the changes for release.

Reply:
Done

Reply:
Can you clarify why the API test rig will not work?

Reply:
automation team is taking lang from this property for prereg

Reply:
It does not make sense to have this property again here, lets remove this property from here and lets talk to the automation team to fix the code from their side.

Reply:
Removed the property @vishwa-vyom